src/transformStateWithClones.js

```diff
@@ -1,13 +1,43 @@
 'use strict';
 
 /**
- * @param {Object} state
+ * @param {Object} stateCopy
  * @param {Object[]} actions
- *
+ *action
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const stateArray = [];
```

Comment:
There are multiple states in the array.
```suggestion
  const statesArray = [];
```
or
```suggestion
  const states = [];
```